perfkitbenchmarker/providers/gcp/flags.py

```diff
@@ -35,3 +35,10 @@
     'https://cloud.google.com/sdk/gcloud/reference/compute/disks/create')
 flags.DEFINE_string('gce_network_name', None, 'The name of an already created '
                     'network to use instead of creating a new one.')
+flags.DEFINE_list('vm_metadata_from_string', [],
+                  'Instance metadata specified through local files. '
+                  'See: --metadata-from-file from '
+                  '`gcloud help compute instances create`.')
+flags.DEFINE_list('vm_metadata', [],
```

Comment:
I know we haven't been consistent about this with the earlier flags, but could you prefix these new ones with 'gcp_'? We've been doing that for other provider-specific flags because the names are global. Alternately, see more general comments I'll post in a bit in the Conversation tab.